go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/go-co-op/gocron v1.28.3
 	github.com/go-errors/errors v1.4.2
 	github.com/go-redis/redis/v8 v8.11.5
-	github.com/golang-jwt/jwt/v4 v4.4.1
+	github.com/golang-jwt/jwt/v4 v4.5.0
```

Comment:
Is this required to get sqlserver support in?

Reply:
That happened automatically when I added the `gorm.io/driver/sqlserver` dependency. That's because they also mention this dependency in their go.sum and we use different versions.

https://github.com/go-gorm/sqlserver/blob/6e2cde390b0a489e530d5771f0885fb138240ba4/go.sum#L10